scripts/build/builders/bouffalolab.py

```diff
@@ -42,11 +42,23 @@ def FlashBundleName(self):
 
 
 class BouffalolabBoard(Enum):
-    BL706_IoT_DVK = 1
-    BL706_NIGHT_LIGHT = 2
+    BL602_IoT_Matter_V1 = 1
+    BL602_IOT_DVK_3S = 2
+    BL602_NIGHT_LIGHT = 3
+    XT_ZB6_DevKit = 101
```

Comment:
There is no reason for this kind of large numeric jump. Just use enum.auto as we do not persist these as numeric.

Reply:
changed